subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/metrics/InstrumentationMetricsSender.kt

```diff
@@ -0,0 +1,26 @@
+package com.avito.instrumentation.metrics
+
+import com.avito.android.stats.CountMetric
+import com.avito.android.stats.GaugeMetric
+import com.avito.android.stats.StatsDSender
+
+class InstrumentationMetricsSender(
+    private val statsDSender: StatsDSender,
+    buildId: String,
+    instrumentationConfigName: String
+) {
+
+    private val prefix = "testrunner.$buildId.$instrumentationConfigName"
+
+    fun sendNotReportedCount(count: Int) {
+        statsDSender.send(prefix, GaugeMetric("status.lost.not-reported", count))
```

Comment:
`buildId.configuration.status.lost.not-reported`
Is status about the whole configuration? It looks strange
Maybe `tests.status.lost` or `tests.stats.lost....` ?

Reply:
chose `tests.status.lost`